packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -21,8 +21,7 @@ export interface AppEventData {
   shop_id: number
   api_client_id: number
   payload: string
-  event_type: string
-  log_type?: string
+  log_type: string
```

Comment:
Should we also rename this interface to `AppLogData` ?

Reply:
Yup. Done.